src/TxHistory/AssetList/AssetList.tsx

```diff
@@ -48,9 +50,11 @@ export const AssetList = (props: Props) => {
         {...props}
         data={sortTokenInfos({wallet, tokenInfos})}
         renderItem={({item: tokenInfo}) => (
-          <Boundary loading={{size: 'small'}} error={{size: 'inline'}}>
-            <AssetItem tokenInfo={tokenInfo} />
-          </Boundary>
+          <ExplorableAssetItem
+            tokenInfo={tokenInfo}
+            balance={balances[tokenInfo.id]}
+            onPress={() => Linking.openURL(config.EXPLORER_URL_FOR_TOKEN(tokenInfo.id))}
```

Comment:
tapping on a token will launch the explorer (we are passing the policyId.assetName) lately fingerprint